Examples/Example-macOS/Source/GTMAppAuthExampleViewController.m

```diff
@@ -65,9 +67,18 @@ @implementation GTMAppAuthExampleViewController
 - (void)viewDidLoad {
   [super viewDidLoad];
 
-  self.keychainStore = [[GTMKeychainStore alloc] initWithItemName:kExampleAuthorizerKey];
+  GTMKeychainAttribute *dataProtection = [GTMKeychainAttribute useDataProtectionKeychain];
+  NSString *bundleID = @"com.example.GTMAppAuth.Example-macOS.test-group";
```

Comment:
Can we grab the bundleID from [`bundleIdentifier`](https://developer.apple.com/documentation/foundation/nsbundle/1418023-bundleidentifier)? Or put it up as a `static const`?